src/components/ListBox.tsx

```diff
@@ -0,0 +1,231 @@
+import {
+  Children,
+  ComponentPropsWithRef,
+  Key,
+  ReactElement,
+  ReactNode,
+  cloneElement,
+  useMemo,
+  useRef,
+} from 'react'
+import { useListBox, useOption } from '@react-aria/listbox'
+import { ListState, useListState } from '@react-stately/list'
+import { Item } from '@react-stately/collections'
+import { useFocusRing } from '@react-aria/focus'
+import { mergeProps } from '@react-aria/utils'
+import { AriaListBoxProps } from '@react-types/listbox'
+import { mergeRefs } from 'react-merge-refs'
+
+import styled, { CSSObject, useTheme } from 'styled-components'
+
+import { Card } from '../index'
+
+export const HEADER_KEY = '$$header$$'
+export const FOOTER_KEY = '$$footer$$'
+
+type ListBoxUnmanagedProps = ComponentPropsWithRef<'div'> & {
+  state: ListState<object>
+  header?: ReactElement
+  footer?: ReactElement
+  headerFixed?: ReactNode
+  footerFixed?: ReactNode
+  extendStyle?: CSSObject
+}
+
+type ListBoxProps = Omit<ListBoxUnmanagedProps, 'state' | 'nextFocusedKeyRef'> & {
+  selectedKey: string
+  onSelectionChange: (key: string) => unknown
+  onHeaderClick?: () => unknown
+  onFooterClick?: () => unknown
+  disallowEmptySelection?: boolean
+  children: ReactElement | ReactElement[]
+}
+
+const ListBoxCard = styled(Card).attrs(() => ({
+  cornerSize: 'medium',
+  hue: 'lighter',
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+}))(_p => ({
```

Comment:
We need to add a `_` as a valid prefix for eslint to ignore unused variables.

Reply:
Updated our eslint config to ignore variables with `_` prefix. You can bump `@pluralsh/eslint-config-typescript` to `v2.3.0` and it will work :slightly_smiling_face:

Reply:
Thanks for making that update! I've removed all these `disable` lines now.